src/current/v22.2/upgrade-cockroach-version.md

```diff
@@ -272,30 +289,26 @@ Once you are satisfied with the new version:
     > RESET CLUSTER SETTING cluster.preserve_downgrade_option;
     ~~~
 
-    {{site.data.alerts.callout_info}}
-    All [schema change](online-schema-changes.html) jobs must reach a terminal state before finalization can complete. Finalization can therefore take as long as the longest-running schema change. Otherwise, the amount of time required for finalization depends on the amount of data in the cluster, as it kicks off various internal maintenance and migration tasks. During this time, the cluster will experience a small amount of additional load.
-    {{site.data.alerts.end}}
+    The series of migration jobs for {{ page.major_version }} runs in sequence.
```

Comment:
The `major_version` variable isn't displaying anything in preview.

```suggestion
    The series of migration jobs for {{ page.version.version }} runs in sequence.
```

Reply:
I fixed the bug, it is working now.